kvmagent/kvmagent/plugins/nvram/nvram_local.py

```
@@ -0,0 +1,185 @@
+                if backing == expected:
+                    dev_name = sys_path.split("/")[-3]  # ex: loop0
+                    return "/dev/%s" % dev_name
+            except IOError:
+                continue
+        return None
+
+    def _make_nvram_to_block_device(self, nvram_install_path):
+        cmd = "losetup --find --show '%s'" % nvram_install_path
+        loop_dev = bash.bash_o(cmd).strip() # type: str
+        if not loop_dev:
+            raise Exception("Failed to setup loop device for %s" % nvram_install_path)
```

Ruff (0.15.2)

[warning] Create your own exception

(TRY002)